cognite/client/_cognite_client.py

```diff
@@ -102,6 +103,7 @@ def __init__(
         self.three_d = ThreeDAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
         self.labels = LabelsAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
         self.relationships = RelationshipsAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
+        self.entity_matching = EntityMatchingAPI(self._config, api_version=self._API_VERSION, cognite_client=self)
```

Comment:
Isn't it already here? https://github.com/cognitedata/cognite-sdk-python/blob/feature/add_entity_matching/cognite/client/beta.py#L9

Reply:
Sorry, I did not know. I got a task to upgrade entity matching from experimental to v1 on dshub. Maybe the supposed version is beta instead of v1. Then I will remove the PR.